cyclonedx_py/client.py

```diff
@@ -282,6 +285,13 @@ def _get_input_parser(self) -> BaseParser:
             input_data = input_data_fh.read()
             input_data_fh.close()
 
+        if self._arguments.input_from_environment:
+            req_names: Optional[List[str]] = None
+            if self._arguments.input_from_requirements:
```

Comment:
@KramNamez This check is redundant since already covered by line 256, right?

Reply:
Ah, yes, the second one is. At that point if we're taking input from the environment, we must also be taking input from requirements.txt. Thanks for catching that.

Reply:
I tried opening a PR again, but a) it wants to pull three commits in again, and b) for some reason it can't just merge it. It's a really simple change: You can just remove the second check and unindent the second line.